docs/user/upgrade.rst

```diff
@@ -90,6 +90,15 @@ Compatibility Matrix
 | Dependencies | N     | N     | Y     |
 +--------------+-------+-------+-------+
 
+Version 3.2.0 Upgrade Guide
++++++++++++++++++++++++++++
+
+Order of Columns in GeoMesa Spark
+---------------------------------
+
+When loading data from a ``SpatialRDDProvider`` in Spark, the ``__fid__`` column has been moved from the first
+element to the last element of the row, in order to better align with other GeoMesa formats.
```

Comment:
Grammarian note:  can we change the split infinitive to "to align better"?;)

Reply:
```suggestion
element to the last element of the row, in order to align better with other GeoMesa formats.
```